lib/msal-common/src/account/AuthToken.ts

```diff
@@ -47,4 +47,27 @@ export class AuthToken {
             throw ClientAuthError.createTokenParsingError(err);
         }
     }
+
+    /**
+     * Determine if the token's max_age has transpired
+     */
+    static checkMaxAge(authTime: number | undefined, maxAge: number): void {
+        /*
+         * per https://auth0.com/docs/authenticate/login/max-age-reauthentication
+         * To force an immediate re-authentication: If an app requires that a user re-authenticate prior to access,
+         * provide a value of 0 for the max_age parameter and the AS will force a fresh login.
+         */
+        if (maxAge === 0) {
```

Comment:
Were there some WAM vs non WAM differences here, @tnorling?

Reply:
IIRC WAM did not support max_age=0 - I'll confirm whether that is still the case

Reply:
It has been resolved.